releases/v1.7.3.toml

```diff
@@ -0,0 +1,45 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+# project_name is used to refer to the project in the notes
+project_name = "containerd"
+
+# github_repo is the github project, only github is currently supported
+github_repo = "containerd/containerd"
+
+# match_deps is a pattern to determine which dependencies should be included
+# as part of this release. The changelog will also include changes for these
+# dependencies based on the change in the dependency's version.
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release of this project for determining changes
+previous = "v1.7.2"
+
+# pre_release is whether to include a disclaimer about being a pre-release
+pre_release = false
+
+# preface is the description of the release which precedes the author list
+# and changelog. This description could include highlights as well as any
+# description of changes. Use markdown formatting.
+preface = """\
+The third patch release for containerd 1.7 contains various fixes and updates.
+
+### Notable Updates
+
+* **RunC: Update runc binary to v1.1.8** ([#8843](https://github.com/containerd/containerd/pull/8843))
+* **CRI: Fix additionalGids: it should fallback to imageConfig.User when securityContext.RunAsUser,RunAsUsername are empty** ([#8824](https://github.com/containerd/containerd/pull/8824))
+* **CRI: write generated CNI config atomically** ([#8825](https://github.com/containerd/containerd/pull/8825))
+* **Move logrus setup code to log package** ([#8831](https://github.com/containerd/containerd/pull/8831))
+* **Cirrus CI: configure apt-get to wait for locks** ([#8814](https://github.com/containerd/containerd/pull/8814))
```

Comment:
Is this a list of all changes in 1.7.2? Some of these don't really seem notable; like this CirrusCI fix that is just about better CI stability and nothing to do with the release.

If I was to make a decision about any others to drop, the "move logrus setup code" and the "Move to CRI-TOOLS v1.27.0" as not really notable; the rest sound like solid bug fixes and maybe worth leaving

Reply:
no, but most of them. I can remove mentioned changes, if they are not really useful.

Reply:
We can remove any notes about CI/Test changes, `ctr` or other tool changes, and dependency updates.

Reply:
dependency updates except for runc binary updates*